backend/repositories/projectRepository.js

```diff
@@ -0,0 +1,33 @@
+var Repository = require('./generalRepository');
+var Project = require('../schemas/projectRepository');

```

Comment:
variable should be called projectRepository. Instances should be written in camelCase + Repository word is required, because Project - is model